gemp-lotr/gemp-lotr-async/src/main/web/src/Game/game.js

```diff
@@ -0,0 +1,20 @@
+var ui;
```

Comment:
extracted inline scripts and styles. Let's hop into a game on playtest and ensure it still works as expected. Any issues would be drastic failures. No edge case failures possible